app/views/layouts/application.html.erb

```diff
@@ -1,19 +1,21 @@
-<!DOCTYPE html>
```

Comment:
Any reason to remove the html5 declaration? I think we should keep this.

Reply:
Good catch! I was deleting some comments and that should not have been  one.